internal/website/content/secrets/vault/_index.md

```diff
@@ -1,4 +0,0 @@
----
```

Comment:
Don't delete this file: this could break `go get`s that operate on the current released version.

Reply:
Done.